models/double_entry_transactions/int_quickbooks__transfer_double_entry.sql

```diff
@@ -23,6 +23,7 @@ transfer_body as (
 final as (
     select 
         transaction_id,
+        row_number() over(partition by transaction_id order by transaction_date) - 1 as index,
```

Comment:
Why do we add the index partition here? Would it make more sense to include it within the `transfer_body` CTE and then bring it in within the `final` CTE as `index`?

Reply:
Similar to how you did it within the `int_quickbooks__bill_payment_double_entry` model.